src/modules/LoggedIn/Project/ProjectForm.tsx

```diff
@@ -5,14 +5,23 @@ import { useProjectSync } from './useProjectSync'
 const ProjectForm = () => {
   const projectSync = useProjectSync()
   const [visible, setVisible] = React.useState<boolean>(false)
+  const [newData, setNewData] = React.useState<any>([
+    {
+      name: '',
+      description: ''
+    }
+  ])
+  const { name, description } = newData
   const toggleModal = React.useCallback(() => {
     setVisible((prev) => !prev)
   }, [])
-
+  const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setNewData({ ...newData, [event.target.name]: event.target.value })
+  }
```

Comment:
di memoize pake useCallback jangan lupa

```suggestion
  const handleChange = React.useCallback(event: React.ChangeEvent<HTMLInputElement>) => {
    setNewData({ ...newData, [event.target.name]: event.target.value })
  }, [])
```